lib/mappers/figma/shadow.js

```diff
@@ -0,0 +1,25 @@
+const { mapColorKeys, tokenize } = require('../../utils');
+const formatColor = require('../sketch/color');
+
+const getShadowEffectProperties = shadow => {
+  const { effects } = shadow.children[0];
+  return {
+    blur: effects[0].offset.radius,
+    x: effects[0].offset.x,
+    y: effects[0].offset.y,
+    type: effects[0].type.toLowerCase().replace('_', '-'),
+  };
+};
+
+const getShadowColor = shadow => {
+  const { color } = shadow.children[0].effects[0];
+  return { color: formatColor(mapColorKeys(color)) };
+};
+
+module.exports = shadow => ({
+  id: shadow.name.split('/').pop(),
+  token: tokenize('shadow'),
+  ...getShadowEffectProperties(shadow),
+  ...getShadowColor(shadow),
+  spread: null, // spread not implemented in figma
```

Comment:
spread is prob. the equiv. of blur in Figma.
Best thing to move forward is to compare outputs side by side for Figma & Sketch

Reply:
I'll check it out.
This was what I found about spread: 
https://spectrum.chat/figma/general/spread-value-in-shadow~148e8d26-7704-40e0-8c67-dbcdedfaaea6